roles/translator/proxy-config.toml

```diff
@@ -18,3 +18,9 @@ min_supported_version = 2
 
 # Minimum extranonce2 size
 min_extranonce2_size = 16
+coinbase_reward_sat = 5_000_000_000
```

Comment:
this will be removed in next PR where we revert the JN to the one that is in the spec (this is the modified proposed and refused version) where coinbase is only managed by pool and not by proxy.